product-service/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "shop-service",
+  "version": "1.0.0",
+  "description": "Serverless aws-nodejs-typescript template",
+  "main": "serverless.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "engines": {
+    "node": ">=14.15.0"
+  },
+  "dependencies": {
```

Comment:
Are those dependencies really needed? Unfortunately, I could not find their usage in the code.